### ISSUE TITLE
[Visual requirement - Bot Framework Web Chat - Card Input]: The visual labels are not defined for all form fields under 'Card Input' screen.

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5344]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:

If a label is not defined for users with visual disabilities and will not get the proper information, then the users will not be able to find out what kind of information is needed in the fields and will face difficulties while navigating on the page.
### Note: 
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Suggested action, Others or unrelated

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label
3. and then select open web chat in new window link at bottom.
4. Navigate to white label experience chat page and type Card Input word in type your message text box.
5. Navigate to the search results and fill details and then navigate to submit results and show card results.
6. Verify whether the visual labels are provided or not.

### What do you expect?

The visual labels should be defined for all form fields under 'Card Input' screen.
**Ex:** for name edit field the visual label should be defined as 'Name'.

### What actually happened?

The visual labels are not defined for all form fields under 'Card Input' screen.
**Observation:** Screen reader (Narrator, nvda, jaws) announcing as 'Name edit'

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/34d6591b-0f92-4bbd-bea4-6a36144fe03d)

### MAS reference

MAS 3.3.2 – Labels or Instructions

### WCAG reference

3.3.2   Labels or Instructions

**Comment by manojs5** [comment]
#A11yMAS;#A11ySev2;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#MAS3.3.2;#Win11-Edge(Chromium);

**Comment by Kirank915** [comment]
@OEvgeny This issue is marked as external to Adaptive cards, could you please confirm if your team will fix the issue, or we need to share this issue to adaptive cards team?

**Comment by OEvgeny** [comment]
This can be fixed by adjusting card payload, this is bot authoring issue. No further action is required.

**Comment by Kirank915** [comment]
@OEvgeny could you please confirm if this issue will be external to the Adaptive cards team. If yes, could you please help us with service tree ID of the Adaptive cards team we will route the bug accordingly.

**Comment by OEvgeny** [comment]
The issue is not in how adaptive cards handle this particular card, rather than in card definition. The card definition can be adjusted, so it doesn't break accessibility. This is something bot author is able to do, but for this particular case, we won't update the test bot (Mockbot) for now.
